### DISCUSSION TITLE
[Feature request] Podcast player & python port

### DISCUSSION THREAD

**Discussion by r57zone** [OnionUI/Onion#1034] [Ideas]
Dear developers, it would be cool to see if it is possible:

Any podcast player (mp3 player , with saving the position of the played, folders and the ability to delete the file). 

A python port so that we can use the device not only for gaming, but for different tasks. A list of python scripts, with the ability to run from the menu and maybe some sort of interface library could be added.

If any of this could be implemented, that would be great. 

Thanks for your hard work.

**Comment by Aemiii91** [comment]
We have a music player - and python 2.7 support.

**Reply by r57zone** [reply]
Found a Python and PyGame in PortCollection, I think I can try to make a podcast player based on this example. Thanks for your reply.

**Comment by bmosley** [comment]
Are there any plans for Python 3.7 or later?

**Comment by SebastianMaung** [comment]
This is an old discussion, but I cross compiled a few things here https://drive.google.com/drive/folders/1DvBGShZKl0NcppL0SlVH6kgFfu_eYYk6?usp=sharing
including Python 3.9 which supports openssl and zlib, and Python 3.10. It also has bash and glibc 2.35, which could open some new paths of development.
